src/components/seo.js

```diff
@@ -25,32 +25,40 @@ function SEO({ description, lang, meta, keywords, title }) {
       meta={[
         {
           name: `description`,
-          content: metaDescription,
+          content: 'A place for Q&A and free mentorship',
+        },
+        {
+          property: `og:title`,
+          content: 'Mentor Night | Free Code Camp',
+        },
+        {
+          property: `og:description`,
+          content: 'A place to be coached or to coach for free along your developer journey',
+        },
+        {
+          property: `og:type`,
+          content: `website`,
+        },
+        {
+          name: `twitter:card`,
+          content: `summary`,
```

Comment:
```suggestion
          name: 'twitter:card',
          content: 'summary',
```

Reply:
We may also want to try out `summary_large_image`. We can try both during the testing phase.